src/test/java/com/exasol/adapter/document/files/AdlsRandomAccessInputStreamTest.java

```diff
@@ -0,0 +1,38 @@
+package com.exasol.adapter.document.files;
+
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.exasol.adapter.document.documentfetcher.files.randomaccessinputstream.RandomAccessInputStream;
+import com.exasol.adapter.document.documentfetcher.files.randomaccessinputstream.RandomAccessInputStreamTestBase;
+import com.exasol.adapter.document.files.adlstestsetup.AdlsTestSetup;
+import com.exasol.adapter.document.files.adlstestsetup.OnlineAdlsTestSetup;
+
+
+class AdlsRandomAccessInputStreamTest extends RandomAccessInputStreamTestBase {
```

Comment:
This looks like an integration test, please rename the class to `...IT`

Reply:
ok, makes sense